lib/puppet/provider/service/systemd.rb

```diff
@@ -14,7 +14,7 @@
   confine :true => Puppet::FileSystem.exist?('/proc/1/comm') && Puppet::FileSystem.read('/proc/1/comm').include?('systemd')
 
   defaultfor :osfamily => [:archlinux]
-  defaultfor :osfamily => :redhat, :operatingsystemmajrelease => ["7", "8"]
+  defaultfor :osfamily => :redhat, :operatingsystemmajrelease => (7..9).to_a
```

Comment:
To be more future-proof, can we do something like the following?

```ruby
defaultfor :osfamily => :redhat
notdefaultfor :osfamily => :redhat, :operatingsystemmajrelease => ["5", "6"]
```

Reply:
Makes sense, these defaults are all very convoluted so it's best to leave it as is :+1: